### DISCUSSION TITLE
Are there plans to support KeyboardAccelerators on WASM?

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno#11419] [Q&A]
What is the current status of ``KeyboardAccelerators`` on WASM? It seems to not be supported, yet. I found issue #3219 that mentions Keyboard accelerators, but does not further discuss it.

Basically I would like this to be supported:

```
<Button Content="Load" Click="Load_Click">
    <Button.KeyboardAccelerators>
        <KeyboardAccelerator Key="L" Modifiers="Control"/>
    </Button.KeyboardAccelerators>
</Button>
```

**Comment by jeromelaban** [comment]
The current status is the same as mentioned in #3219, it's on the backlog, but do not have an ETA for it.

**Reply by TopperDEL** [reply]
Thanks!
